cmake/GrpcTargets.cmake

```diff
@@ -126,5 +126,10 @@ function(add_grpc_library NAME)
   add_library(${NAME} STATIC ${generated_sources})
   target_compile_options(${NAME} PUBLIC -Wno-unused-parameter)
   target_include_directories(${NAME} SYSTEM PUBLIC ${include_paths})
-  target_link_libraries(${NAME} PUBLIC userver-grpc Protobuf)
+  find_package(absl)
+  if(absl_FOUND)
```

Comment:
You should check the grpc library version and search for absl if the version is greater than some value (they started requesting absl quite recently)